### DISCUSSION TITLE
Documentation about kernel options at install, especially regarding configuration of NICs and VIP needs to be expanded

### DISCUSSION THREAD

**Discussion by red-lichtie** [siderolabs/talos#7861] [Ideas]
In the installation documentation, there should be a mention of the ability to configure the kernel options before installing a node, especially with regards to ethernet adapters.

I followed the [Proxmox guide](https://www.talos.dev/v1.5/talos-guides/install/virtualized-platforms/proxmox/).

In various examples "eth0" is used as the NIC device name but in fact, the default image uses predictable network interface names for the devices.

The documentation should indicate that controlplane.yaml and worker.yaml can be edited before the installation, especially with regards to `extraKernelArgs`.

e.g.
```yaml
machine:
    install:
        extraKernelArgs:
            - net.ifnames=0
```

In the [Virtual (shared) IP guide](https://www.talos.dev/v1.5/talos-guides/network/vip/) there is also no mention of `eth0` not being the default device name.

The guide should warn that the device name **will** be something other than `eth0` and that they can use `talosctl dmesg --nodes ${CONTROLPLANE_NODE_NAME | grep eth0}` to see what the name was changed to.

It would be nice to have the device name(s) in the summary dashboard beside the IP address(es).

**Comment by smira** [comment]
https://github.com/siderolabs/talos/pull/7864

**Comment by red-lichtie** [comment]
Merged
